### PR TITLE
Pass down include and exclude as kwarg to repr_mimebundle

### PR DESCRIPTION
**Pull request by Carreau**
Followup on #10526

### CODE REVIEW

**Comment by rgbkrk** [comment]
Didn't even know about these until now. 👍

**Review by rgbkrk — APPROVED** [review]

File: IPython/core/formatters.py [hunk 1]
```diff
@@ -937,6 +937,35 @@ def _check_return(self, r, obj):
...
         return r
 
+    @catch_format_error
+    def __call__(self, obj, include=None, exclude=None):
+        """Compute the format for an object.
+
+        Identical to parent's method but we pass extra parameters to the method.
+
+        Unlike other _repr_*_ `_repr_mimebundle_` should allow extra kwargs, in
+        particular `include` and `exclude`.
```
Comment (rgbkrk) [inline IPython/core/formatters.py]: 👍

**Review by rgbkrk — APPROVED** [review]

**Comment by Carreau** [comment]
@meeseeksdev backport

File: IPython/core/formatters.py [hunk 2]
```diff
@@ -937,6 +937,35 @@ def _check_return(self, r, obj):
...
+                pass
+            else:
+                return printer(obj)
+            # Finally look for special method names
+            method = get_real_method(obj, self.print_method)
+
+            if method is not None:
+                d = {}
+                d['include'] = include
+                d['exclude'] = include
```
Comment (Carreau) [inline IPython/core/formatters.py]: That was a wrong copy past. fixed in #10527 hopefully.

**Pull request merged**